### PR TITLE
Wiki describing how to add transaction

### PR DESCRIPTION
**Pull request by nithinkrishna**

### CODE REVIEW

File: docs/Orchestrator.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+```
+
+2) Admin invokes `addTransaction` with the destination contract address and `bytes`
+as encoded from step 1.
+
+### Current list of transactions
+
+1. UniswapV2 (AMPL-ETH): sync() => `0x08a564924C26D8289503bbaA18714B9C366dF9a5`, `0xfff6cae9`
+2. UniswapV2 (AMPL-USDC): sync() => `0xc5be99A02C6857f9Eac67BbCE58DF5572498F40c`, `0xfff6cae9`
+3. Balancer Bpool (AMPL-ETH): gulp() => `0xeefb11cA05c6F0d5252757Dbe35fFC4458108e89`, `0x8c28cbe8000000000000000000000000d46ba6d942050d489dbd938a2c909a5d5039a161`
```
Comment (ahnaguib) [inline docs/Orchestrator.md:32]: please add the unencoded token parameter here as well

File: docs/Orchestrator.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }],
+}, ['0xD46bA6D942050d489DBd938a2C909A5d5039A161']);
+```
+
+2) Admin invokes `addTransaction` with the destination contract address and `bytes`
+as encoded from step 1.
+
+### Current list of transactions
+
+1. UniswapV2 (AMPL-ETH): sync() => `0x08a564924C26D8289503bbaA18714B9C366dF9a5`, `0xfff6cae9`
```
Comment (brandoniles) [inline docs/Orchestrator.md:30]: This is what I get for the current transactions array:
0. UniV2 DAI-AMPL => 0x08a564924C26D8289503bbaA18714B9C366dF9a5
1. UniV2 ETH-AMPL => 0xc5be99A02C6857f9Eac67BbCE58DF5572498F40c
2. Balancer ETH-AMPL => 0xeefb11cA05c6F0d5252757Dbe35fFC4458108e89

Comment (nithinkrishna) [reply docs/Orchestrator.md:30]: Is this not what we have here?

File: docs/Orchestrator.md (hunk 1 above)
Comment (nithinkrishna) [reply docs/Orchestrator.md]: 👍

**Pull request merged**